index.html

```diff
@@ -505,7 +506,7 @@ <h3>Information for Authors</h3>
         This is often a more convenient way of providing keyboard navigation within widgets, such as a <rref>listbox</rref>, where the widget occupies only one stop in the page <kbd>Tab</kbd> sequence and other keys, typically arrow keys, are used to focus elements inside the widget.
       </p>
 	    <p>Typically, the author will use host language <a>semantics</a> to put the widget in the <kbd>Tab</kbd> sequence (e.g., <code>tabindex="0"</code> in HTML) and <code>aria-activedescendant</code> to point to the ID of the currently active descendant. The author, not the user agent, is responsible for styling the currently active descendant to show it has keyboard focus. The author cannot use <code>:<span class="css-selector">focus</span></code> to style the currently active descendant since the actual focus is on the container.</p>
-	    <p>More information on managing focus can be found in the <a href="#keyboard" class="practices">Developing a Keyboard Interface</a> section of the <cite><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</cite>.</p>
+	    <p>More information on managing focus can be found in the <a href="practices/keyboard-interface" class="practices">Developing a Keyboard Interface</a> section of the <cite><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</cite>.</p>
```

Comment:
This relative path seems problematic. It won't resolve on github.io or w3.org as far as I can tell.

Reply:
Interested. I would have sworn It did resolve in https://w3.org/TR/2024/WD-wai-aria-1.3-20240118/

Then I'd say we can use the absolute path for safety. https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/

Given that the practices are no longer in TR space we may want to drop them from the spec URLs section.

Reply:
Oh, sorry I had missed that connection! Thanks for clarifying!